ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2800,6 +2800,55 @@ def test_numpy_instance_array_wrap__(
     )
 
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="take",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"), num_arrays=1
+    ),
+)
+@given(
+    indices=st.one_of(
+        st.integers(),
+        st.lists(st.integers()),
+        st.tuples(st.integers()),
+        st.arrays(dtype=np.int64, shape=st.tuples(st.integers())),
+    ),
+    axis=st.one_of(st.none(), st.integers()),
+    mode=st.one_of(st.just("raise"), st.just("wrap"), st.just("clip")),
+)
```

Comment:
I'm not really sure why you are using `@given` can you instead write all of your strategies in `@handle_function_method`?

Reply:
yes, sure i will write the stratigies in  handle_function_method